planning/motion_velocity_planner/autoware_motion_velocity_planner_common/src/utils.cpp

```diff
@@ -200,4 +201,23 @@ double calc_possible_min_dist_from_obj_to_traj_poly(
     object_possible_max_dist;
   return possible_min_dist_to_traj_poly;
 }
+
+double get_dist_to_traj_poly(
+  const geometry_msgs::msg::Point & point,
+  const std::vector<autoware_utils::Polygon2d> & decimated_traj_polys)
+{
+  const auto point = autoware_utils_geometry::Point2d(point.x, point.y);
+
+  double dist_to_traj_poly = std::numeric_limits<double>::infinity();
+
+  for (const auto & decimated_traj_poly : decimated_traj_polys) {
+    const double current_dist_to_traj_poly =
+      boost::geometry::distance(decimated_traj_poly, boost_point);
```

Comment:
My suggestion was partial. Please see your code entirely.

```suggestion
      boost::geometry::distance(decimated_traj_poly, point);
```

Reply:
All done now